src/server.test.ts

```diff
@@ -3,8 +3,10 @@ import { parseMarkdown } from "./server";
 test("renders an empty doc", () => {
   const ast = parseMarkdown("");
 
-  expect(ast.toJSON()).toEqual({
-    content: [{ type: "paragraph" }],
-    type: "doc",
-  });
+  if ( ast ) {
```

Comment:
in which case it's falsy?

Reply:
this is just to satisfy typescript i don't know to be honest lol